### PR TITLE
Provide more info for Apache-2.0 application

### PR DESCRIPTION
**Pull request by char0n**
Ref #871

### CODE REVIEW

**Comment by mlinksva** [comment]
Thanks. @ShaneCurcuru has occasionally commented in this repo. Shane, if you have any thoughts on/review of adding this link to ASF policy around LICENSE and NOTICE files, would be much appreciated. Not asking for an official statement. 😄

File: _licenses/apache-2.0.txt [hunk 1]
```diff
@@ -7,7 +7,7 @@ hidden: false
 
 description: A permissive license whose main conditions require preservation of copyright and license notices. Contributors provide an express grant of patent rights. Licensed works, modifications, and larger works may be distributed under different terms and without source code.
 
-how: Create a text file (typically named LICENSE or LICENSE.txt) in the root of your source code and copy the text of the license into the file.
+how: Create a text file (typically named LICENSE or LICENSE.txt) in the root of your source code and copy the text of the license into the file. Consider adding a NOTICE text file with explicit Copyright Notice as well. More information about the license application is available in Apache Foundations [Assembling LICENSE and NOTICE files](https://infra.apache.org/licensing-howto.html) document.
```
Comment (waldyrious) [inline _licenses/apache-2.0.txt]:
```suggestion
how: Create a text file (typically named LICENSE or LICENSE.txt) in the root of your source code and copy the text of the license into the file. Consider adding a NOTICE text file with an explicit copyright notice as well. More information about the license application is available in Apache Foundation's [Assembling LICENSE and NOTICE files](https://infra.apache.org/licensing-howto.html) document.
```

**Comment by char0n** [comment]
Suggestion from @waldyrious accepted.

**Comment by mlinksva** [comment]
Looking at this again, I've decided against merging. The doc doesn't correspond entirely with the added text, and is a bit confusing even for me:

- There's no instruction to add a copyright notice; it's just in the example at the end
-  It's unclear whether notices for bundled code are supposed to be added to LICENSE or NOTICE
- Requests for help are directed to an ASF committee, which doesn't seem appropriate for general consumption

A NOTICE file is not essential and I don't know that these instructions are helpful for people who don't already know OSS licenses well and are choosing one, ie audience for the site.

Thanks for the idea and feel free to tell me why I should reconsider.

**Comment by ShaneCurcuru** [comment]
Apologies for the delay in review.

NOTICE files are only required in section 4.4 if the work you're then redistributing already contains a NOTICE file, so I agree this is a somewhat esoteric point to include for the targeted audience, who you're just trying to get to use **a** license, any one.

Also the various pages on apache.org about NOTICE are primarily very specific ASF policy for Apache projects, and is pretty heavy reading for someone outside the ASF just trying to use the license.

I agree this is not needed, given the audience.

Bigger picture (separately from this issue), I have two suggestions (that I'd help with if I weren't doing a new startup!):

- Having more resource links to _understanding_ software licenses would be a useful addition, beyond what's in the About - which is also a  little buried - having a top-level nav to "Understanding Licenses" or the like would be cool.
- Adding a FAQ section to each license might be useful.  It couldn't be definitive, but there are some common questions about some licenses that you could link to, even just to get users aware of the question.  For example, a common question for Apache and many copyleft licenses are: "Which other licenses is this compatible with?" Where you could link to the relevant [ASF](https://www.apache.org/legal/resolved.html#criteria) and [FSF](https://www.gnu.org/licenses/license-list.html#SoftwareLicenses) pages.

**Comment by joseliaon** [comment]
Vladimir?

**Comment by char0n** [comment]
Sure, sounds like a start. Original problem that is was trying to solve is - where to put Copyright Notice when using Apache 2.0 license. In majority cases, people tend to put the copyright notice in the text of the Apache 2.0 license, which is incorrect application of the license and breaks  its "Reusable without rewording" principle.

**Pull request closed without merging**